### PR TITLE
Twitter cards

### PR DESCRIPTION
**Pull request by alehel**
## Description

Currently Twitter cards don't contain a lot of information. This PR improves the look of these cards.

**Twitter card with current code**
image

**Twitter card with this PR**
image

You define the image used in the card by defining the socialImage tag in the md file for a page. If it hasn't been defined, we fall back to using your photo.jpg file. The twitter card description uses the existing description tag in the md file. 

image

I've also updated the sample md files with the socialImage tag.

## Related Issues

Related to #350

### CODE REVIEW

**Comment by alxshelepenok** [comment]
Wow, this is a very good solution.
It remains to fix a couple of linting errors

**Comment by alehel** [comment]
I forgot to update the tests. I've pushed a quick fix, but I must admit. I don't have a lot of experience writing such tests. Be nice if you could take a look @alxshelepenok

**Comment by alehel** [comment]
I'm a little confused by the linting errors. I'm currently getting the following for the [Layout.js](https://github.com/alehel/blog/blob/twitter-cards/src/components/Layout/Layout.js) file,

`16:17  error  Expected a line break after this opening brace`

If you look at [Sidebar.js](https://github.com/alehel/blog/blob/master/src/components/Sidebar/Sidebar.js) from the master branch it seems to be formatted the same, but without triggering an error. So I'm a little confused what I've done wrong.

**Comment by alxshelepenok** [comment]
@alehel if the number of props is more than 3
```
const Layout = ({
  children,
  title,
  description,
  socialImage
}: Props) => {
```

**Comment by alehel** [comment]
@alxshelepenok ah, I see. Thanks, I'll fix it now.

I tried running 'yarn lint' before I pushed these changes. However I never got any of these errors. Just a bunch of errors about the linebreak being CRLF, which gets fixed by git anyway. Is there something I need to do to get this command to run correctly on Windows?

**Comment by alxshelepenok** [comment]
@alehel strange, my preffered os is Fedora Linux, so I didn’t notice it.
To fix these errors, you can try adding `linebreak-style: "unix"` to eslint config.

**Comment by alehel** [comment]
@alxshelepenok thanks, I got linting working locally by adding

` "linebreak-style": ["error", "windows"],`

to the config file. I'm now getting the expected linting errors. I guess the large number of CRLF errors were preventing the actual issues from surfacing in the terminal. 

Looks like all linting issues are fixed now.

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
+  description,
+  socialImage
+}: Props) => {
+  const { author, url } = useSiteMetadata();
+  const metaImage = socialImage != null ? socialImage : author.photo;
+  const metaImageUrl = url + withPrefix(metaImage);
+
+  return (
+    <div className={styles.layout}>
+      <Helmet>
```
Comment (alxshelepenok) [inline src/components/Layout/Layout.js]: I also recommend adding an image for open graph
```<meta property="og:image" content={metaImageUrl} />```

Comment (alehel) [reply src/components/Layout/Layout.js]: Fixed 👍

**Comment by alxshelepenok** [comment]
Excellent thank you!

**Pull request merged**